IyiKuyoro/Ride-My-Way
# #158818209 Add a database cleanup after all to tests

### What does this PR do?
Delete all test data added to the db
#### Description of Task to be completed?
The following endpoints test have been affected:
`/api/v1/rides/:rideId/requests`
`/api/v1/users/rides`
#### How should this be manually tested?
Clone the repository first and start the server on a terminal with `npm run start`
  * Run tests with `npm run test`
  * Check db to ensure data added by the test is not present
#### Any background context you want to provide?
As a quick fix to the test data residing permanently on the database, I decided to delete them after every test.
#### What are the relevant pivotal tracker stories?
  #158818209
